src/main/java/org/miracum/streams/fhirtoserver/SendToServerProcessor.java

```diff
@@ -133,16 +133,19 @@ public Consumer<Resource> sink() {
         return;
       }
 
-      MDC.put(
-          "bundleFirstEntryId",
-          bundle
-              .getEntryFirstRep()
-              .getResource()
-              .getIdElement()
-              .toUnqualifiedVersionless()
-              .toString());
-      MDC.put(
-          "bundleFirstEntryType", bundle.getEntryFirstRep().getResource().getResourceType().name());
+      if (((Bundle) resource).hasEntry()) {
+        MDC.put(
+            "bundleFirstEntryId",
+            bundle
+                .getEntryFirstRep()
+                .getResource()
+                .getIdElement()
+                .toUnqualifiedVersionless()
+                .toString());
+        MDC.put(
+            "bundleFirstEntryType",
+            bundle.getEntryFirstRep().getResource().getResourceType().name());
```

Comment:
vielleicht also weiter oben

`var firstEntryResource = bundle.getEntryFirstRep() =! null then .getResource()`

Reply:
oh mann, dass mir das nicht aufgefallen ist. da shätte jetzt wieder einen ewigen feedback loop benötigt. vielen dank

Reply:
```java
var firstEntryResource = bundle.getEntryFirstRep().getResource();
      if (firstEntryResource != null) {
        MDC.put(
            "bundleFirstEntryId",
            firstEntryResource.getIdElement().toUnqualifiedVersionless().toString());
        MDC.put("bundleFirstEntryType", firstEntryResource.getResourceType().name());
      }
```
Würdest du etwa `getFirstEntryRep()` auch noch auf NULL prüfen?

Reply:
klingt nach nem unit test :D. aber muss jetzt erstmal nicht blocken daran, wenn ich "The first repetition of repeating field [entry](https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-structures-r4/org/hl7/fhir/r4/model/Bundle.html#entry), creating it if it does not already exist" https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-structures-r4/org/hl7/fhir/r4/model/Bundle.html#getEntryFirstRep() richtig verstehe dann selbst wenn es null ist erzegt er ein neues.  Also der check auf getResource reicht wohl